web/src/components/Cleanup.vue

```diff
@@ -185,9 +185,10 @@ v-layout.cleanup-wrapper(row)
 
 <style lang="scss">
 @mixin masked() {
-  background-color: var(--v-secondary-lighten1);
+  background-color: var(--v-secondary-lighten3);
+  box-shadow: inset 0 0 0 .5px var(--v-secondary-lighten2);
   font-weight: 300;
-  color: var(--v-secondary-darken1);
+  color: var(--v-secondary-base) !important;
```

Comment:
I think this one can be replaced if `tr.datarow, tr.datarow.header` is added at the very end.